Lib/test/test_site.py

```diff
@@ -519,6 +520,23 @@ def test_license_exists_at_url(self):
 class StartupImportTests(unittest.TestCase):
 
     def test_startup_imports(self):
+        # Get sys.path in isolated mode (python3 -I)
+        popen = subprocess.Popen([sys.executable, '-I', '-c',
+                                  'import sys; print(repr(sys.path))'],
+                                 stdout=subprocess.PIPE,
+                                 encoding='utf-8')
+        stdout = popen.communicate()[0]
+        self.assertEqual(popen.returncode, 0, repr(stdout))
+        isolated_paths = eval(stdout)
+
+        # bpo-27807: Even with -I, the site module executes all .pth files
+        # found in sys.path (see site.addpackage()). Skip the file if at least
+        # one .pth file is found.
+        for path in isolated_paths:
+            pth_files = glob.glob(os.path.join(path, "*.pth"))
+            if pth_files:
```

Comment:
```suggestion
            if pth_files := glob.glob(os.path.join(path, "*.pth"))
```

I don't know whether you prefer this expression, but it can be reduced as the one line..
But not the necessary thing it's up to you!

Reply:
I'm old school and like to have assignment on its own line :-) There are cases where it enhances readability IMO, but here I prefer two lines... I'm not sure that I can explain why :-)

Reply:
> I'm old school and like to have assignment on its own line :-)

Got it!